knowledge-base/actions/golfzaptw/action-auto-reviews-from-branches/action-security.yml

```diff
@@ -0,0 +1,9 @@
+name: 'Auto reviews'
+github-token:
+  action-input: 
+    input: GITHUB_TOKEN
+  permissions: 
```

Comment:
I think permission here should be set to `write` to create review for PR's, view [here](https://github.com/golfzaptw/action-auto-reviews-from-branches/blob/269a54ff625c35873ba83b807b5530c89d37d0e9/index.js#L68)